pyflakes/checker.py

```diff
@@ -1557,6 +1597,15 @@ def CALL(self, node):
                 node.func.attr == 'format'
         ):
             self._handle_string_dot_format(node)
+
+        if (
+            _is_typing(node.func, 'cast', self.scopeStack) and
+            len(node.args) >= 1 and
```

Comment:
`typing.cast` takes exactly two arguments. Is the `>= 1` for robustness?

Reply:
Yeah -- it's to guard the `node.args[0]` below, rather than as part of the "is this `typing.cast` check.